modules/ROOT/pages/dw-core-functions-matches.adoc

```diff
@@ -13,7 +13,7 @@ Checks if an expression matches the entire input string.
 
 For use cases where you need to output or conditionally process the matched
 value, see
-https://docs.mulesoft.com/mule-runtime/4.3/dataweave-pattern-matching[Pattern Matching in DataWeave].
```

Comment:
TODO: This was changed to "latest" in the latest source.

* https://docs.mulesoft.com/mule-runtime/latest/dataweave-pattern-matching[Pattern Matching in DataWeave].